redaxo/src/core/lib/error_handler.php

```diff
@@ -60,43 +60,20 @@ public static function handleException($exception, $showTrace = true)
         rex_response::setStatus($status);
 
         if (rex::isSetup() || rex::isDebugMode() || ($user = rex_backend_login::createUser()) && $user->isAdmin()) {
-            // TODO add a beautiful error page with usefull debugging info
-            $buf = '';
-            $buf .= '<pre>';
-            $buf .= '"' .  get_class($exception) . '" thrown in ' . $exception->getFile() . ' on line ' . $exception->getLine() . "\n";
-            if ($exception->getMessage()) {
-                $buf .= '<b>' . ($exception instanceof ErrorException ? self::getErrorType($exception->getSeverity()) . ': ' : '') . $exception->getMessage() . "</b>\n";
-            }
-
-            $cause = $exception->getPrevious();
-            while ($cause) {
-                $buf .= "\n";
-                $buf .= 'caused by ' . get_class($cause) . ' in ' . $cause->getFile() . ' on line ' . $cause->getLine() . "\n";
-                if ($cause->getMessage()) {
-                    $buf .= '<b>' . ($cause instanceof ErrorException ? self::getErrorType($cause->getSeverity()) . ': ' : '') . $cause->getMessage() . "</b>\n";
-                }
-
-                $cause = $cause->getPrevious();
-            }
-
-            if ($showTrace) {
-                $buf .= "\n";
-                $buf .= $exception->getTraceAsString();
-            }
-
             if (!rex::isSetup() && rex::isBackend() && !rex::isSafeMode()) {
-                $buf .= "\n\n";
-                $buf .= '<a href="' . rex_url::backendPage('packages', ['safemode' => 1]) . '">activate safe mode</a>';
+                //$buf .= '<a href="' . rex_url::backendPage('packages', ['safemode' => 1]) . '">activate safe mode</a>';
             }
-
-            $buf .= '</pre>';
-        } else {
-            // TODO small error page, without debug infos
-            $buf = 'Oooops, an internal error occured!';
+            $whoops = new \Whoops\Run;
```

Comment:
Ich denke, der Parameter kann weg. Der wurde ja nur auf `false` gesetzt, wenn ein Fatal Error über die shutdown-Function abgefangen wurde.
Mit PHP 7 habe ich den Fall gerade gar nicht simulieren können. Kommt das da noch vor?
Mit PHP 5.6 hingegen kann man den Fall simulieren, indem man irgendwo `foo();` einfügt.
Jedoch sieht dann mit dem Stacktrace in whoops alles ok aus. Somit kann der Parameter weg, denke ich.